### DISCUSSION TITLE
Woensdag 09-10-2024

### DISCUSSION THREAD

**Discussion by Annevd** [rutgerkock/buurtcampus-oost#5] [Stand-ups]
- Hoe gaat het met je?
- Wat wil je deze week bereiken (aan werken)?
- Waar loop je tegen aan?

**Comment by Nazneen05x** [comment]
- **Hoe gaat het met je?**

Het kan beter! nog niet helemaal mijzelf! Ik ben morgen wel aanwezig bij de sprint review, vind ik belangrijk. Ik hoop dat ik volgende week weer normaal naar school kan komen.

- Wat wil je deze week bereiken (aan werken)?

Deze week ben ik bezig geweest met het bouwen van de geveltuin pagina. De pagina is af, ik moet nu nog alleen de pagina responsief maken. Morgen ben ik verplan om de agenda-carousel en geveltuin pagina te laten zien aan de opdrachtgever!

**Issues:**

[#37](https://github.com/users/Nazneen05x/projects/14/views/1?pane=issue&itemId=82435059&issue=Nazneen05x%7CS4-S19-Bieb-In-Bloei%7C37)

[#38](https://github.com/users/Nazneen05x/projects/14/views/1?pane=issue&itemId=82435530&issue=Nazneen05x%7CS4-S19-Bieb-In-Bloei%7C38)

[#39](https://github.com/users/Nazneen05x/projects/14/views/1?pane=issue&itemId=82435781&issue=Nazneen05x%7CS4-S19-Bieb-In-Bloei%7C39)

[#40](https://github.com/users/Nazneen05x/projects/14/views/1?pane=issue&itemId=82436213&issue=Nazneen05x%7CS4-S19-Bieb-In-Bloei%7C40)

[#41](https://github.com/users/Nazneen05x/projects/14/views/1?pane=issue&itemId=82436410&issue=Nazneen05x%7CS4-S19-Bieb-In-Bloei%7C41)

- Waar loop je tegen aan?

Op dit moment nog niks, het bouwen van de pagina gaat goed op dit moment! 😊
